go.mod

```diff
@@ -84,7 +84,7 @@ require (
 	go.uber.org/zap v1.27.0
 	go.viam.com/api v0.1.351
 	go.viam.com/test v1.1.1-0.20220913152726-5da9916c08a2
-	go.viam.com/utils v0.1.110
+	go.viam.com/utils v0.1.112-0.20241101173204-42f7f8eb50e1
```

Comment:
What's up with this weird pseudoversion? Maybe try `GOPROXY=direct go get -u go.viam.com/utils`?

Reply:
I upgraded goutils to `HEAD` without releasing a new version initially, but now there is a proper version out so this is fixed: https://github.com/viamrobotics/rdk/pull/4511/commits/f9d7211f14f8206c268ff97902178a46879c25dd